sentences/RMB.js

```diff
@@ -1,44 +1,75 @@
 /*
 Heading and distance to waypoint:
-$IIRMB,A,x.x,a,,,IIII.II,a,yyyyy.yy,a,x.x,x.x,x.x,A,a*hh
- I I I I I I I I I_Speed to WP in knots
- I I I I I I I I_True heading to destination in degrees
- I I I I I I I_Distance to destination in miles
- I I I I I_ ___ I_Longitude of the WP to destination, E/W
- I I I__ I_Latidude of the WP to destination, N/S
- I I_Direction of cross-track error, L/R
- I_Distance of cross-track error in miles
+                                                             14
+        1 2   3 4    5    6       7 8        9 10  11  12  13|  15
+        | |   | |    |    |       | |        | |   |   |   | |   |
+ $--RMB,A,x.x,a,c--c,c--c,llll.ll,a,yyyyy.yy,a,x.x,x.x,x.x,A,m,*hh<CR><LF>
+------------------------------------------------------------------------------
+
+Field Number:
+
+1. Status, A= Active, V = Void
+2. Cross Track error - nautical miles
+3. Direction to Steer, Left or Right
+4. TO Waypoint ID
+5. FROM Waypoint ID
+6. Destination Waypoint Latitude
+7. N or S
+8. Destination Waypoint Longitude
+9. E or W
+10. Range to destination in nautical miles
+11. Bearing to destination in degrees True
+12. Destination closing velocity in knots
+13. Arrival Status, A = Arrival Circle Entered
+14. FAA mode indicator (NMEA 2.3 and later)
+15. Checksum
 */
-// to verify
+
 const nmea = require('../nmea.js')
 module.exports = function (app) {
+
+  const apiVersion = app.config.version ? parseInt(app.config.version.split('.')[0]) : 1
```

Comment:
Now that I have my brains actually turned on: I don't think this is the correct fix. You can not simply look at the server's version and decide which paths are going to be used. @panaaj can you verify if this is correct?

I see a few options going forward:
- create a copy of RMB, call it RMB2.js and put the API version 2 path based conversion there as a separate conversion the user can activate
- add all the paths as keys to this file (with defaults), keep track of which paths are getting updates and use them in the conversion. this will allow smooth transition with no user action needed when input switches to v2

Reply:
> You can not simply look at the server's version and decide which paths are going to be used.

Yes and no.... Course API was introduced in Signalk server v2.0.0 BUT the implementation could still be using v1 paths due to incoming data still being mapped to v1 SignalK paths.